packages/web/src/components/nav/desktop/PlaylistLibrary/CreatePlaylistLibraryItemButton.tsx

```diff
@@ -106,18 +106,11 @@ export const CreatePlaylistLibraryItemButton = (props: Props) => {
             ref={anchorRef}
             iconLeft={IconSave}
             onClick={() => handleClickPill(onClick)}
+            css={{ height: '18px' }}
```

Comment:
just a reminder height: 18 works

Reply:
This PR will fully cement in my brain that emotion CSS is smarter than css modules.